sonar-plugin-api/src/main/java/org/sonar/api/measures/CountDistributionBuilder.java

```diff
@@ -33,7 +33,9 @@
  * <p>An example of usage : you wish to record the number of violations for each level of rules priority
  *
  * @since 1.10
+ * @deprecated since 5.6. Scanner side is not responsible to aggregate measures since 5.2.
  */
+@Deprecated
 public class CountDistributionBuilder implements MeasureBuilder {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)